<launch>
  <arg name="CAMERA" default="/head_resized" />
  <arg name="IMAGE_TYPE" default="image_rect_throttle" />
  <arg name="RESIZE_RATE" default="0.25" />
  <arg name="COMPRESS_TYPE" default="compressed" />

  <node ns="$(arg CAMERA)"
        name="screenpoint_view" pkg="image_view2" type="image_view2"
        output="screen" >
    <remap from="image" to="$(arg IMAGE_TYPE)" />
    <remap from="$(arg CAMERA)/$(arg IMAGE_TYPE)/screenpoint"
           to="/pointcloud_screenpoint_nodelet/point" />
    <remap from="/tf" to="/tf_dummy" />
    <param name="autosize" value="false" />
    <param name="image_transport" value="$(arg COMPRESS_TYPE)" />
    <param name="resize_scale_x" value="$(arg RESIZE_RATE)" />
    <param name="resize_scale_y" value="$(arg RESIZE_RATE)" />
  </node>

</launch>
